atd-etl/afd_ems_import/afd_incident_upload.py

```diff
@@ -111,7 +112,12 @@ def get_secrets():
             "opitem": "RDS Bastion Key",
             "opfield": ".private key",
             "opvault": VAULT_ID,
-            },
+        },
+        "record_age_maximum_in_days": {
```

Comment:
I think it would be great to be able to update the number of days that the script looks back from 1PW but let me know if anyone sees a reason not to add this parameter.

Reply:
I think this is a great idea too. I dont remember how long we were getting empty csvs, and being able to change the maximum without having to change the code is great.

Reply:
I like this idea!

Reply:
👍 from me, I think it makes a lot of sense.